### ISSUE TITLE
Some entity previews can't be interacted with

### ISSUE THREAD

**Issue by Abbe98** [OpenRefine/OpenRefine#5605]
Reconciliation entity previews shows when you are either hovering the reconciliated item's link or the preview popup.

Sometimes a reconciliation entity previews can end up next to the edit-cell button making it impossible to hover/interact with the popup without causing the popup to close first, because of the edit button which sits atop of the link:

![Screenshot from 2023-02-04 12-26-32](https://user-images.githubusercontent.com/2631719/216764826-3e3dff6d-1ba4-4e33-b1a9-8348aac2627d.png)

**Comment by thadguidry** [comment]
Whoa, what happened to the `<style font-size>` ???
Double check the HTML returned by the preview service?  It's up to them still, yes?
Honestly, we need reconcile previews to be a bit more extensible for customized viewport scenarios to accomodate any users/services needs.  Like GND might probably like to have a much better customized viewport for their reconciling.

And I and others have always wanted **full page rendering in another tab on another screen** through some mechanism like:
1. Local Storage Events
2. Broadcast Channel API
3. Service Worker Post Message
4. Window Post Message

Alas.

**Comment by Abbe98** [comment]
> Whoa, what happened to the <style font-size> ???
Double check the HTML returned by the preview service? It's up to them still, yes?

It does not have an effect on the issue, it's all about the positioning and the logic for when to display the preview. One can reproduce the issue with the Wikidata and Wikimedia Commons endpoints as well:

![Screenshot from 2023-02-04 13-13-25](https://user-images.githubusercontent.com/2631719/216767068-eefabf46-015d-4c4e-befd-9fea0fb1062e.png)

**Comment by wetneb** [comment]
Yes the positioning of those previews can be extremely frustrating. It would be great to see work on this.

**Comment by Abbe98** [comment]
@thadguidry I think a potential solution is to introduce some delay in between the cursor leaving one of the areas and the popup being hidden. It would also allow one to move the cursor a little more freely when intending to interact with the popup.

**Comment by tfmorris** [comment]
I think this is a regression, but I'm not sure from how long ago. I certainly don't remember them being this fiddly.

One thing which would help is to center the popup vertically on the relevant item. That would relax the mouse tracking requirements for the user which can be tricky with the current top alignment.
